builder/docker.go

```diff
@@ -163,21 +164,35 @@ func (b *Builder) getDockerMounts() ([]string, error) {
 // RunCommandInContainer will pull the content necessary to build a docker
 // image capable of running the desired command, build that image, and then
 // run the command in that image.
-func (b *Builder) RunCommandInContainer(cmd []string) error {
-	format, err := b.getUpstreamFormat(b.UpstreamVer)
-	if err != nil {
-		return err
+func (b *Builder) RunCommandInContainer(cmd []string, offline bool) error {
```

Comment:
Why introduce a new `offline` variable? `docker.go` is within the `builder` package, so it can already access the package-wide `Offline` variable that gets set when `--offline` is passed. (See [`getUpstreamBundles`](https://github.com/clearlinux/mixer-tools/blob/master/builder/builder.go#L414) for how it's used elsewhere.) Do you actually have cases where you want to pass a value of `offline` here that is different from what `builder.Offline` will hold?